recipes/flaightidl/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "flaightidl" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f19969361b2167fc349a8ddef97833006083ebc7a39eef69f4968d6c987f7f89
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - protobuf >=3.5.0,<4.0.0
+    - python
+  run:
+    - protobuf >=3.5.0,<4.0.0
+    - python
+
+test:
+  imports:
+    - flyteidl
+    - flyteidl.admin
+    - flyteidl.core
+    - flyteidl.datacatalog
+    - flyteidl.event
+    - flyteidl.plugins
+    - flyteidl.plugins.sagemaker
+    - flyteidl.service
+    - validate
+
+about:
+  home: https://latch.bio 
```

Comment:
This URL is not working. Also, is there a link to a git repo you could point to in `dev_url` ?

Reply:
Hey there - thanks for the review! I'm not sure what you mean by the URL isn't working - I copied it into my browser and it seemed to work just fine. Would you mind expanding on that? Thank you!

Reply:
Also, the github repo our code is hosted on is private, so its currently not possible to provide a link - will talk to my team about this and get back to you.

Reply:
When I went to that link yesterday, I got "File Not found" as a string in my browser window. Seems to be fine now, though.

Reply:
It's fine if you don't have a public dev_url.